decidim-admin/app/views/decidim/admin/participatory_processes/show.html.erb

```diff
@@ -30,11 +30,22 @@
     :slug,
     :short_description,
     :description
-    %>
+  %>
+
   <dt><%= display_label(@participatory_process, :hero_image) %></dt>
-  <dd><%= image_tag @participatory_process.hero_image.url.to_s %></dd>
+  <dd>
+    <% if @participatory_process.hero_image %>
+      <%= image_tag @participatory_process.hero_image.url.to_s %>
```

Comment:
**NAIS COLORS**. Just updated the PR appending `?` to uploaders, which seems to be the standard-yet-ugly way to deal with this in carrierwave.